web-ng/src/app/components/project-list/project-list.component.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright 2021 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<div fxLayout="column">
+  <gnd-header-layout> </gnd-header-layout>
+
+  <h1 fxFlex class="recent-projects">Recent Projects</h1>
```

Comment:
IIuc this is a list of all available project, not just recent ones? If so please rename accordingly.

Reply:
changed to All Projects.